driver-core/src/test/java/com/datastax/driver/core/ConsistencyTest.java

```diff
@@ -15,716 +15,340 @@
-        }
-)
-public class ConsistencyTest extends AbstractPoliciesTest {
+
+import static com.datastax.driver.core.TestUtils.nonQuietClusterCloseOptions;
+import static org.testng.Assert.*;
+
+public class ConsistencyTest {
+
 
     private static final Logger logger = LoggerFactory.getLogger(ConsistencyTest.class);
+    private ScassandraCluster sCluster;
 
-    private Cluster.Builder tokenAwareRoundRobin() {
-        return Cluster.builder()
-                .withSocketOptions(new SocketOptions().setReadTimeoutMillis(300000))
-                .withLoadBalancingPolicy(new TokenAwarePolicy(new RoundRobinPolicy()));
-    }
+    @BeforeClass
+    public void setUp() {
+        sCluster = ScassandraCluster.builder().withNodes(1).build();
+        sCluster.init();
 
-    private Cluster.Builder tokenAwareRoundRobinNoShuffle() {
-        return Cluster.builder()
-                .withSocketOptions(new SocketOptions().setReadTimeoutMillis(300000))
-                .withLoadBalancingPolicy(new TokenAwarePolicy(new RoundRobinPolicy(), false));
     }
 
-    private Cluster.Builder tokenAwareRoundRobinDowngrading() {
-        return Cluster.builder()
-                .withSocketOptions(new SocketOptions().setReadTimeoutMillis(300000))
-                .withLoadBalancingPolicy(new TokenAwarePolicy(new RoundRobinPolicy()))
-                .withRetryPolicy(DowngradingConsistencyRetryPolicy.INSTANCE);
+    @AfterMethod
+    public void tearDown() {
+        clearActivityLog();
     }
 
-    private Cluster.Builder tokenAwareRoundRobinNoShuffleDowngrading() {
-        return Cluster.builder()
-                .withSocketOptions(new SocketOptions().setReadTimeoutMillis(300000))
-                .withLoadBalancingPolicy(new TokenAwarePolicy(new RoundRobinPolicy(), false))
-                .withRetryPolicy(DowngradingConsistencyRetryPolicy.INSTANCE);
-    }
 
-    private Cluster.Builder roundRobinDowngrading() {
-        return Cluster.builder()
-                .withSocketOptions(new SocketOptions().setReadTimeoutMillis(300000))
-                .withLoadBalancingPolicy(new RoundRobinPolicy())
-                .withRetryPolicy(DowngradingConsistencyRetryPolicy.INSTANCE);
+    public void clearActivityLog() {
+        for (Scassandra node : sCluster.nodes()) {
+            node.activityClient().clearAllRecordedActivity();
+        }
     }
 
-    private Cluster.Builder tokenAwareDCAwareRoundRobinNoShuffleDowngrading() {
+    public Cluster.Builder builder() {
+        //Note: nonQuietClusterCloseOptions is used to speed up tests
         return Cluster.builder()
-                .withSocketOptions(new SocketOptions().setReadTimeoutMillis(300000))
-                .withLoadBalancingPolicy(new TokenAwarePolicy(DCAwareRoundRobinPolicy.builder().withLocalDc("dc2").build(), false))
-                .withRetryPolicy(DowngradingConsistencyRetryPolicy.INSTANCE);
+                .addContactPoints(sCluster.address(1).getAddress())
```

Comment:
It looks liek for some reason an NPE is thrown at this line.   I think it might have to do with `@BeforeClass` not targeting group `short` (although not sure why that would matter).    It might not be something that shows up unless you run with the surefire runner.  Looking into it.